Package.swift

```diff
@@ -14,9 +14,14 @@ let package = Package(
     ],
     dependencies: [],
     targets: [
-        .binaryTarget(
+        .target(
             name: "RadarSDK",
-            path: "dist/RadarSDK.xcframework"
+            path: "RadarSDK",
+            exclude: ["Info.plist", "RadarSDK.h"],
+            publicHeadersPath: "Include",
+            cSettings: [
+                .headerSearchPath("Internal")
+            ]
```

Comment:
Can you add comments in here about why these things are included or excluded?

Reply:
> The choice of Internal was mine, though.

Note that I reverted the `Internal/` move in a later commit, because I found a workaround.